### DISCUSSION TITLE
Any way to have autoplay?

### DISCUSSION THREAD

**Discussion by darknebulae** [cifkao/html-midi-player#68] [Q&A] [answered]
Loop is great but I would love if I could autoplay the files aswell, is this possible?

**Comment by cifkao** [comment, marked as answer]
Most modern browsers do not support audio autoplay. Audio must be started by a user action.

In Chrome, you get a message like this if you try to start audio right after the page loads:
```
The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu
```

So this is not really possible.

**Comment by DonDonos78** [comment]
Hello how to automatically start playing even having in mind that no all browser support it?
how to set it up?

**Reply by cifkao** [reply]
You can call `player.start()` to start playback.
